index.html

```diff
@@ -2209,14 +2209,25 @@ <h5>Character naming template</h5>
 </aside>
 
-<p>Some notes about the markup used. The <code translate="no" class="kw">bdi</code> element is used to ensure that example characters that are right-to-left do not interfere with the layout of the page. The <code translate="no" class="kw">lang</code> attribute should be filled in appropriately to get the correct font selection for a given context. Examples in East Asian languages (such as Chinese, Japanese, or Korean) or in the Arabic script can sometimes require greater care in choosing a language tag.</p>
+<p>The <code translate="no" class="kw">bdi</code> element is used to ensure that example characters that are right-to-left do not interfere with the layout of the page.</p>
 
-<aside class="example" title="Code point sequences">
-	<p>Short sequences of characters (2 or 3 code points) should list the character names, separated by unstyled <kbd> + </kbd>. For example: <span class="codepoint" translate="no"><bdi lang="ar" dir="rtl">&#x062B;&#x064E;</bdi><span class="uname">U+062B ARABIC LETTER THEH</span> + <span class="uname">U+064E ARABIC FATHA</span></span></p>
-	<p>Long sequences will sometimes just list the code points, although the character names should be retained where possible for clarity. An example can be found in this doucment in the <a href="#char_term_def">discussion of the composed "family" emoji</a>: <span class="codepoint"><bdi lang="en">👨‍👩‍👧‍👧</bdi> <span class="uname">U+1F468 U+200D U+1F469 U+200D U+1F467 U+200D U+1F467</span></span></p>
-</aside>
+<p>The <code translate="no" class="kw">lang</code> attribute should be filled in appropriately to get the correct font selection for a given context. Examples in East Asian languages (such as Chinese, Japanese, or Korean) or in the Arabic script can sometimes require greater care in choosing a language tag.</p>
+
+<p>Short sequences of characters should list the character names, separated by <span class="codepoint" translate="no">+</span>.
+
+<aside class="example" title="Example of a code point sequence">
+	<p>For example, this example: <span class="codepoint" translate="no"><bdi lang="ar" dir="rtl">&#x062B;&#x064E;</bdi><span class="uname">U+062B ARABIC LETTER THEH</span> + <span class="uname">U+064E ARABIC FATHA</span></span> uses the following markup:</p>
 
+<pre>
+&lt;span class="codepoint" translate="no">&lt;bdi lang="ar" dir="rtl">&amp;#x062B;&amp;#x064E;&lt;/bdi>
+   &lt;span class="uname">U+062B ARABIC LETTER THEH&lt;/span> + 
+   &lt;span class="uname">U+064E ARABIC FATHA&lt;/span>
+&lt;/span>
+</pre>	
+
+</aside>
 
+<p>Long sequences will sometimes just list the code points, although the character names should be retained where possible for clarity. An example can be found in this doucment in the <a href="#char_term_def">discussion of the composed "family" emoji</a>: <span class="codepoint"><bdi lang="en">👨‍👩‍👧‍👧</bdi> <span class="uname">U+1F468 U+200D U+1F469 U+200D U+1F467 U+200D U+1F467</span></span></p>
```

Comment:
Having slept on this... I think the thing to do here is to move the margins to the `bdi` element. People will naturally include a space before/after the uname span when omitting the exemplar character (just as they will put a space before/after the codepoint span all of the time). Instead of stage managing the space before the uname, we can manage the space around the example character instead.

Check out the latest push on this PR. I've removed all of the spaces between `bdi` and `span` and styled the `bdi` with 0.5em after margin (recall that the font is at 130%). Standalone uses of `uname` in the doc look normal. We might tweak the size/space a bit on the `bdi`? Thoughts?

Reply:
Moving the margin to the bdi is certainly the right thing to do - sorry i forgot to mention that earlier.

However, if the bdi contains RTL text the different amounts of spacing won't appear in the expected location, relative to the uname.  So i think you need margin-inline:.5em;

For my own documents, I'm also thinking about extensions which allow you to add a class name which will add extra space to the left or right of the exemplar character.  This can be useful when dealing with combining marks - some of which may overlap the uname or the preceding text.  I also remove the spacing altogether when including an image, rather than a character, since the images i'm using usually have space built in (because they are square).